pkgs/development/python-modules/treex/default.nix

```diff
@@ -31,6 +31,11 @@ buildPythonPackage rec {
     ./relax-deps.patch
   ];
 
+  postPatch = ''
```

Comment:
Note that there's already `relax-deps.patch` above that does the same for other dependencies. I have no strong preferences RE: the way to do it (= fine to use `postPatch` + `substituteInPlace`) but would like to avoid using both methods simultaneously. If you prefer `postPatch` - mind migrating the deps from the patch above and deleting the patch?

Reply:
ah sorry, missed that